### DISCUSSION TITLE
整理阿里读光的表格识别模型

### DISCUSSION THREAD

**Discussion by SWHL** [RapidAI/RapidOrientation#8] [Ideas]
- 考虑到现有表格识别模型较弱，尝试阿里读光下的表格识别模型效果。
  - [读光-表格结构识别-有线表格](https://www.modelscope.cn/models/damo/cv_dla34_table-structure-recognition_cycle-centernet)
  - [读光-表格结构识别-无线表格
  ](https://www.modelscope.cn/models/damo/cv_resnet-transformer_table-structure-recognition_lore/summary)
- 如果效果满足，会将其整合到现有仓库下，欢迎有兴趣的小伙伴一起做。

**Comment by nissansz** [comment]
效果怎么样？

**Comment by SWHL** [comment]
参考链接中有在线测试demo，请自行尝试查看效果，我这里暂未来得及评估

**Comment by nissansz** [comment]
这个模型有离线onnx文件吗？

**Comment by nissansz** [comment]
怎么转换成onnx然后在rapidstructure使用啊？

**Comment by SWHL** [comment]
采用torch.export.onnx转换，整理前后处理，就可以集成到rapid-structer中了

**Comment by nissansz** [comment]
可以帮忙转换一下吗？不会转换啊

**Comment by SWHL** [comment]
不会可以学啊，我也不是本来就会的

**Comment by nissansz** [comment]
要安装完整的modelscope环境吧？这个环境都安装不成功

**Comment by nissansz** [comment]
https://www.modelscope.cn/models/damo/cv_dla34_table-structure-recognition_cycle-centernet/files

pytorch_model.pt文件没法下载，你下载了这个文件没有？

**Comment by nissansz** [comment]
从codelab下载了pt模型，但是不知道怎么转成onnx

**Comment by nissansz** [comment]
转换好模型了吗？

**Comment by SWHL** [comment]
> 转换好模型了吗？

日期不定。本来就是为爱发电。如果着急，你可以自己去转。

**Comment by SWHL** [comment]
已经整理完毕，详情移步：[TableStructureRec](https://github.com/RapidAI/TableStructureRec)
